test/tokenbridge.test.ts

```diff
@@ -47,6 +47,7 @@ describe('TokenBridge', async function() {
   let l2tokenB: StarknetContract;
   let TokenBridgeL2: StarknetContractFactory;
   let tokenBridgeL2: StarknetContract;
+  let rewAaveToken: StarknetContract;
```

Comment:
We should probably have a consistent naming between L1 and L2. We could rename `rewAaveToken` with `rewAaveTokenL2`, and `rewAaveL1` with `rewAaveTokenL1`.

Reply:
fixed